client/devdocs/design/playground-scope.js

```diff
@@ -0,0 +1,111 @@
+export FormSettingExplanation from 'components/forms/form-setting-explanation';
+export FormStateSelector from 'components/forms/us-state-selector';
+export FormTelInput from 'components/forms/form-tel-input';
+export FormTextarea from 'components/forms/form-textarea';
+export FormTextInput from 'components/forms/form-text-input';
+export FormTextInputWithAction from 'components/forms/form-text-input-with-action';
+export FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+export FormToggle from 'components/forms/form-toggle';
+export Gauge from 'components/gauge';
+export GlobalNotices from 'components/global-notices';
+export Gravatar from 'components/gravatar';
+export HeaderCake from 'components/header-cake';
+export HeaderButton from 'components/header-button';
+export ImagePreloader from 'components/image-preloader';
+export InfoPopover from 'components/info-popover';
+export InputChrono from 'components/input-chrono';
+export JetpackColophon from 'components/jetpack-colophon';
+export JetpackLogo from 'components/jetpack-logo';
+export LanguagePicker from 'components/language-picker';
+export ListEnd from 'components/list-end';
+export Main from 'components/main';
+export Notices from 'components/notice';
+export Pagination from 'components/pagination';
+export PaymentLogo from 'components/payment-logo';
+export PieChart from 'components/pie-chart';
+export Popovers from 'components/popover';
+export ProgressBar from 'components/progress-bar';
+export Ranges from 'components/forms/range';
+export Rating from 'components/rating';
+export Ribbon from 'components/ribbon';
+export ScreenReaderText from 'components/screen-reader-text';
+export Search from 'components/search';
+export SearchCard from 'components/search-card';
+export SectionHeader from 'components/section-header';
+export SectionNav from 'components/section-nav';
+export NavTabs from 'components/section-nav/tabs';
+export NavSegmented from 'components/section-nav/segmented';
+export NavItem from 'components/section-nav/item';
+export SegmentedControl from 'components/segmented-control';
+export SelectDropdown from 'components/select-dropdown';
+export ShareButton from 'components/share-button';
+export SiteTitleControl from 'components/site-title';
+export Spinner from 'components/spinner';
+export SpinnerButton from 'components/spinner-button';
+export SpinnerLine from 'components/spinner-line';
+export SplitButton from 'components/split-button';
+export Suggestions from 'components/suggestions';
+export TextDiff from 'components/text-diff';
+export TileGrid from 'components/tile-grid';
+export TimeSince from 'components/time-since';
+export Timezone from 'components/timezone';
+export TokenFields from 'components/token-field';
+export Tooltip from 'components/tooltip';
+export Version from 'components/version';
+export VerticalMenu from 'components/vertical-menu';
+export Wizard from 'components/wizard';
```

Comment:
Noting that `eslint` complains about this for me:

![image](https://user-images.githubusercontent.com/87168/39055525-23bbeaf4-44bd-11e8-9e18-43f9fc099d02.png)

I'm surprised that CI tests didn't fail for that? Could it be just my editor?

Reply:
Oh actually take that back, looks like it's not `eslint` but Atom's own [ts](https://atom.io/packages/ide-typescript) gone mad. :-)